app/forms/concerns/decidim/reporting_proposals/form_builder_override.rb

```diff
@@ -12,6 +12,7 @@ module FormBuilderOverride
       included do
         def file_field(object_name, options = {})
           return super(object_name, options) unless Decidim::ReportingProposals.use_camera_button
+          return super(object_name, options) if @object_name == "editor_image" || @object_name == "oauth_application"
```

Comment:
```suggestion
        alias_method :original_file_field, :file_field
        
        def file_field(object_name, options = {})
          return super(object_name, options) unless Decidim::ReportingProposals.use_camera_button
          return original_file_field unlesss @template.respond_to?(:snippets)
```

Hi @laurajaime, I think the best approach would just do something like this